sanitize.go

```diff
@@ -0,0 +1,50 @@
+package inputsanitizer
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const sanitize_replacement_string string = "REDACTED"
+
+var sensitiveFieldNames = []string{
+	`account[_|/-|\s](.[\w\s_/-]+)*key`,
+	`api[_|/-|\s](.[\w\s_/-]+)*key`,
+	`auth[_|/-|\s](.[\w\s_/-]+)*key`,
+	`azurerm-account-key`,
+	`client[_|/-](.[\w\s_/-]+)*key`,
+	`contrasena`,
+	`contraseña`,
```

Comment:
what are these for?

Reply:
although there shouldn't be 2 so i'll remove one line